pilot/pkg/networking/core/v1alpha3/cluster_builder_test.go

```diff
@@ -205,54 +236,63 @@ func TestApplyDestinationRule(t *testing.T) {
 
 	for _, tt := range cases {
 		t.Run(tt.name, func(t *testing.T) {
-			instances := []*model.ServiceInstance{
-				{
-					Service:     tt.service,
-					ServicePort: tt.port,
-					Endpoint: &model.IstioEndpoint{
-						Address:      "192.168.1.1",
-						EndpointPort: 10001,
-						Locality: model.Locality{
-							ClusterID: "",
-							Label:     "region1/zone1/subzone1",
+			if tt.name == "destination rule with pass through subsets" {
```

Comment:
this was meant for testing I imagine?

btw, the new goland  allows selectively running individual table driven tests if you use it - pretty useful

Reply:
Yes. Meant for testing, forgot to remove.